crates/model/src/solver_competition.rs

```diff
@@ -77,6 +77,9 @@ pub struct CompetitionAuction {
 pub struct SolverSettlement {
     pub solver: String,
     pub objective: Objective,
+    #[serde(flatten)]
+    pub score: Score, // auction based score
```

Comment:
Why do we distinguish between the different score types in the settlement? Do we need to share this information (i.e. share with other solvers how their competition are computing scores)?